suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -418,3 +423,14 @@ export const initDevices = createThunk(
         }
     },
 );
+
+export const createImportedAccountThunk = createThunk(
```

Comment:
Desktop will be implementing portfolio tracker in the future as well. It makes sense to keep this logic here I think. There are thunks in common that we don't use either. Should they be in desktop only then? The point of common is to have logic that can be used between BOTH packages. Only because it's not used now doesn't mean it can't be used later. To me, this makes sensse and is not platform specific.